bonfire/Modules/Guides/Libraries/GuideCollection.php

```diff
@@ -147,9 +147,21 @@ public function pageLinks()
         $currentPage = current_url();
         $currentPage = substr($currentPage, strpos($currentPage, 'guides/'. $this->alias) + $offset);
 
+		$count_sub_folders = count(
+			array_filter(
+				$pages ,
+				function($k) {
+					//If the key is a string, it is a subfolder
+					if(!is_numeric($k)) {
+						return $k;
+					}
+				},
+				ARRAY_FILTER_USE_KEY)
+		);
+
```

Comment:
I think I fixed the error in the GuideCollection library when calculating the previous and next pages: the subfolders had to be removed from the count ...